apps/multiplatform/common/src/desktopMain/kotlin/chat/simplex/common/DesktopApp.kt

```diff
@@ -31,10 +30,31 @@ import java.io.File
 val simplexWindowState = SimplexWindowState()
 
 fun showApp() = application {
+  val state = getStoredWindowState() // creates file if not exists; comes with proper defaults
+
   // For some reason on Linux actual width will be 10.dp less after specifying it here. If we specify 1366,
   // it will show 1356. But after that we can still update it to 1366 by changing window state. Just making it +10 now here
-  val width = if (desktopPlatform.isLinux()) 1376.dp else 1366.dp
-  val windowState = rememberWindowState(placement = WindowPlacement.Floating, width = width, height = 768.dp)
+  val windowState: WindowState = rememberWindowState(
+    placement = WindowPlacement.Floating,
+    width = state.width.dp,
+    height = state.height.dp,
+    position = WindowPosition(state.x.dp, state.y.dp)
+  )
+
+  LaunchedEffect(
+    windowState.position.x.value,
+    windowState.position.y.value,
+    windowState.size.width.value,
+    windowState.size.height.value
+  ) {
```

Comment:
will it still work if you just use 
```kotlin 
KeyChangeEffect(windowState) {
...
}
```
? It should, because WindowPositionSize is a data class.